static/js/chart/draw.ts

```diff
@@ -829,6 +829,7 @@ function drawStackBarChart(
  * @param unit
  */
 function drawGroupBarChart(
+  containerElement: HTMLDivElement,
```

Comment:
What's `containerElement` for?

Reply:
Before, updating the charts was done by finding the div within the tile with the correct id (old line 857 in this file). This no longer works because tile divs are now hidden under a shadow root. Instead, we pass in the div to update directly as `containerElement`. If you know of a better way to do this, please do share and I can update!

Reply:
That sounds good- maybe we can drop the id parameter all together in a future PR